modules/cardinal_service_migrate/src/Plugin/migrate/process/CustomSkipEmptyDivs.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
@mdyoung3 I appreciate the effort level here, but I would just ask Laryn to fix the feed so if there aren't any opportunities, it doesn't send us empty HTML. This is meant to be minimal effort and we shouldn't be asked to manipulate the data that comes in to this degree, especially for this Opportunities field. We should be asking Laryn's team to modify the feed to our requirements as much as possible.

Reply:
@joegl I removed the plugin and sent an email to Laryn to remove any empty divs.